src/shared/url.ts

```diff
@@ -38,6 +38,8 @@ export const api = {
     base: `/api/kube/apis/servicecatalog.k8s.io/v1beta1`,
     create: (namespace = "default") =>
       `${api.serviceinstances.base}/namespaces/${namespace}/serviceinstances`,
+    delete: (namespace = "default") =>
+      `${api.serviceinstances.base}/namespaces/${namespace}/serviceinstances`,
```

Comment:
I think we're moving away from this url object and instead just defining the URLs in the ServiceCatalog classes, so it's safe to not add this here (doesn't look like it's being used)

Reply:
Removed, no longer used